tests/includes/class-test-query.php

```diff
@@ -265,6 +265,29 @@ public function test_comment_activitypub_object() {
 		$this->assertNull( Query::get_instance()->get_activitypub_object() );
 	}
 
+	/**
+	 * Test user at URL activity object.
+	 *
+	 * @covers ::get_activitypub_object
+	 */
+	public function test_user_at_url_activity_object() {
+		$user_id = $this->factory->user->create(
```

Comment:
```suggestion
		$user_id = self::factory()->user->create(
```
Does your IDE have access to wp unit test files to show this as [deprecated](https://github.com/WordPress/wordpress-develop/blob/6.7.2/tests/phpunit/includes/abstract-testcase.php#L29)? `self::factory()` is the intended replacement.